litmus-portal/frontend/src/views/Community/GeoMap/styles.ts

```diff
@@ -0,0 +1,62 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  binarySwitch: {
+    paddingBottom: theme.spacing(2),
+  },
+  cityMapMarkerStyles: {
+    fill: theme.palette.map.cityMap.marker.fill,
+    '& circle': {
+      stroke: theme.palette.map.cityMap.marker.stroke,
+      strokeWidth: '2',
+      strokeLinecap: 'round',
+      strokeLinejoin: 'round',
+      transform: 'translate(-12, -24)',
+      cx: '12',
+      cy: '10',
```

Comment:
these numeric values dosen't need to be in quotes

Reply:
Thank you for your feedback :+1: